http/src/main/resources/init-resources/jupyter-docker-compose.yaml

```diff
@@ -2,19 +2,19 @@
 # configuring memory options in container mode. See discussion in:
```

Comment:
Wonder if we should name this file `jupyter-docker-compose-dataproc.yaml`

Reply:
there's so much yml, bash changes in this PR and it's a bit time consuming to debug...so I'm inclined to save this renaming to another time :)